### PR TITLE
RF | _OrganizationFolderTest> testCreateOrganizationFolderNavigation

### PR DESCRIPTION
**Pull request by RusAli**

### CODE REVIEW

**Comment by RusAli** [comment]
[RF | _OrganizationFolderTest> testCreateOrganizationFolderNavigation](https://trello.com/c/QblJbZIU/1112-rf-organizationfoldertest-testcreateorganizationfoldernavigation)

File: src/test/java/_OrganizationFolderTest.java [hunk 1]
```diff
@@ -159,28 +129,43 @@ public void createOrganizationFolderEmptyName() {
...
+    public void testProjectsTab() {
+        boolean actualResult = new HomePage(getDriver())
+                .clickNewItem()
+                .setProjectName(TestUtils.getRandomStr())
+                .setProjectTypeOrganizationFolder()
+                .clickOkAndGoToConfig()
+                .clickProjectTab()
+                .ckeckChildProjectIsDisplayed();
+
+        Assert.assertTrue(actualResult);
```
Comment (ke-de-ve) [inline src/test/java/_OrganizationFolderTest.java:142]: add second parameter - message to show if assertion failed.

   * Asserts that a condition is true. If it isn't, an AssertionError, with the given message, is thrown.
   *
   * @param condition the condition to evaluate
   * @param message   the assertion error message
   */
  public static void assertTrue(boolean condition, String message) {

**Review by ke-de-ve — APPROVED** [review]

**Review by EugeniaSrednitskaya — APPROVED** [review]

File: src/test/java/_OrganizationFolderTest.java (hunk 1 above)
Comment (azellus-tercia) [reply src/test/java/_OrganizationFolderTest.java:142]: assertTrue() is a common method of Assert. Do you mean to override it?

Comment (RusAli) [reply src/test/java/_OrganizationFolderTest.java:142]: @ke-de-ve @azellus-tercia 
As i suppose, TestNG has method   
```
public static void assertTrue(boolean condition, String message) {
    if (!condition) {
      failNotEquals(condition, Boolean.TRUE, message);
    }
```

**Pull request merged**